driver.js

```diff
@@ -498,22 +497,36 @@ class Driver {
     }
   }
 }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Deze kunnen wel weg toch?

Reply:
Ik vind ze wel fijn, om duidelijk `Driver` van `Site` te onderscheiden. Ik heb geprobeerd ze in losse bestanden te stoppen, maar `wappalyzer.js` werkt zo raar, dat kan niet zomaar.